ONSdigital/response-operations-ui
# added show password field to sign-in page

# What and why?
Unlike frontstage/elsewhere on response-ops, the sign-in page on response-ops did not have an option to show the password. This PR fixes it to make it more accessible and consistent.

# How to test?
Access response-ops and check that the show password option is there and that it works. Also, run acceptance tests.

N.B 
This was extended to remove attributes from all password fields and correctly display errors, because of that activate-account and change password also need testing as well as the login page
# Trello
[Card](https://trello.com/c/FRHWS9Sc)
